broker/src/main/java/io/zeebe/broker/clustering/atomix/AtomixFactory.java

```diff
@@ -114,14 +114,14 @@ private static RaftPartitionGroup createRaftPartitionGroup(
             .withFlushOnCommit();
 
     // by default, the Atomix max entry size is 1 MB
-    final ByteValue maxMessageSize = networkCfg.getMaxMessageSize();
-    partitionGroupBuilder.withMaxEntrySize((int) maxMessageSize.toBytes());
+    final int maxMessageSize = (int) networkCfg.getMaxMessageSizeInBytes();
+    partitionGroupBuilder.withMaxEntrySize(maxMessageSize);
 
     Optional.ofNullable(dataCfg.getLogSegmentSize())
```

Comment:
Should we also have a `getLogSegmentSizeInBytes` method here as well then?

Reply:
Done